packages/eth2.0-state-transition/src/epoch/registryUpdates.ts

```diff
@@ -34,7 +34,7 @@ export function processRegistryUpdates(config: IBeaconConfig, state: BeaconState
   const activationQueue = state.validators.filter((validator) =>
     isEligibleForActivation(config, state, validator)
     // Order by the sequence of activation_eligibility_epoch setting and then index
-  ).sort((a, b) => (a.activationEligibilityEpoch - b.activationEligibilityEpoch) || 1);
+  ).sort((a, b) => (a.activationEligibilityEpoch - b.activationEligibilityEpoch) || -1);
```

Comment:
should this be changed?

Reply:
Yes according to docs, negative value makes a receive smaller index (moves to the front of array). This caused failed mainnet spec test on master.

Reply:
That said, I think we need validator index (or at least original index) because I doubt this is bubble sort where this would make sense